### ISSUE TITLE
expose sklearn/lale utils for data loading and manipulation

### ISSUE THREAD

**Issue by ppalmes** [IBM/Lale.jl#18]
- train-split, metric, crossval, datasets
- https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html
